201-api-management-create-with-internal-vnet-application-gateway/prereqs/.settings.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
remove this - I don't see anything that requires this to be in the same RG

Reply:
Done